.gitattributes

```diff
@@ -1 +1,2 @@
-* text=auto
\ No newline at end of file
+* text=auto
+*.cmd text eol=crlf
```

Comment:
>cmd text eol=crlf [](http://example.com/codeflow?start=2&length=17)

nit: remove extra trailing blank line 3

Reply:
ack, will fix when we decide on correct approach.

Reply:
Please recheck and reopen if necessary.